components/script/dom/document.rs

```diff
@@ -3780,10 +3780,12 @@ impl Document {
         self.animations.borrow()
     }
 
-    pub(crate) fn update_animations_post_reflow(&self) -> AnimationsUpdate {
-        self.animations
-            .borrow_mut()
-            .do_post_reflow_update(&self.window, self.current_animation_timeline_value())
+    pub(crate) fn update_animations_post_reflow(&self, js_runtime: *mut JSRuntime) {
```

Comment:
Let's add a getter to Window for the JS runtime instead so we don't need to pass around an unsafe pointer.

Reply:
:+1: